modules/ctrl_dbus/baresipbus.h

```diff
@@ -64,6 +72,13 @@ void dbus_baresip_emit_event (
     const gchar *arg_evtype,
     const gchar *arg_param);
 
+void dbus_baresip_emit_message (
+    DBusBaresip *object,
+    const gchar *arg_ua,
+    const gchar *arg_peer,
+    const gchar *arg_ctype,
+    const gchar *arg_body);
+
```

Comment:
the diff from the generated file is very large to review :)

Reply:
Yes, this is also generated. So it is a good suggestion to generate through the build.